wpacceptance.json

```diff
@@ -0,0 +1,17 @@
+{
+  "name": "tenup-theme-scaffold",
+  "exclude": [
+    "node_modules"
+  ],
+  "tests": [
+      "./tests/wpa/*.php"
+  ],
+  "enforce_clean_db": false,
+  "project_path": "%WP_ROOT%/wp-content/themes/theme-scaffold",
+  "environment_instructions": [
+    [
+      "install wordpress where url is http://tenupthemescaffold.test",
```

Comment:
The scaffolding script has a set of string patterns it uses for this https://github.com/10up/project-scaffold/blob/master/10upScaffold.js#L65 can we update this to `http://tenup-scaffold.test` so the CLI will catch it?

Reply:
Updated, thanks for the call out.